src/directory-sync/interfaces/directory-group.interface.ts

```diff
@@ -19,3 +19,25 @@ export interface DirectoryGroupResponse {
   updated_at: string;
   raw_attributes: any;
 }
+
+export interface EventDirectoryGroup<TRawAttributes = any> {
```

Comment:
This was the existing `WebhookDirectoryGroup` type, but I took it back into the `directory-sync` folder since it didn't make sense for this to live in the `common` folder.